fatips/300.md

```diff
@@ -0,0 +1,438 @@
+           "mvnansuvbeybqyyyewvyqv273212oi3onbvttbywtybhxzzxuv",
+       ]
+      }
+   ]
+}
+```
+
+(note: extIds shown are not realistic values)
+
+<br/>
+
+### `get-balance` :
+
+Get the balance of an address for a token
+
+#### Parameters:
+
+| Name         | Type   | Description                               | Validation                      | Required |
+| ------------ | ------ | ----------------------------------------- | ------------------------------- | -------- |
+| `fa-address` | string | The Factoid address to get the balance of | Must be a valid Factoid address | Y        |
+
+#### Response:
+
+```json
+{  
+   "jsonrpc":"2.0",
+   "id":1,
+   "result": 2321111
+}
+```
+
+<br/>
+
+### `get-stats` :
+
+Get overall statistics for a token
+
+#### Parameters:
+
+| Name | Type | Description | Validation | Required |
+| ---- | ---- | ----------- | ---------- | -------- |
+|      |      |             |            |          |
+
+#### Response:
+
+```json
+{  
+   "jsonrpc":"2.0",
+   "id":1,
+    "result":{
+        "supply":10000000,
+        "circulatingSupply":8113,
+        "transactions":18429,
+        "issuanceTimestamp": 1518286500,
+        "lastTransactionTimestamp": 1539212767
+    }
+}
+```
+
+
+
+### `get-nf-token` :
+
+Get a non fungible token by ID. The token belong to non fungible token class.
+
+#### Parameters:
+
+| Name          | Type   | Description                             | Validation                                 | Required |
+| ------------- | ------ | --------------------------------------- | ------------------------------------------ | -------- |
+| `nf-token-id` | string | The unique ID of the non fungible token | Must be a valid issued non fungible token. | Y        |
+
+#### Response:
+
+```json
+{  
+   "jsonrpc":"2.0",
+   "id":1,
+    "result":{
+    	"id": "0",
+        "metadata":{
+            "name": "Mr. Cucumber"
+        }
+    }
+}
+```
+
+
+
+### `send-transaction`:
+
+Send A FAT transaction to a token
+
+#### Parameters:
+
+| Name     | Type   | Description                                                  | Validation                                                   | Required |
+| -------- | ------ | ------------------------------------------------------------ | ------------------------------------------------------------ | -------- |
+| `tx`     | object | The fully formed & signed FAT transaction JSON               | Must conform to all transaction validation criteria of the token's spec | Y        |
+| `extids` | array  | The array of hex encoded ed25519 signatures & RCDs (extids) for each transaction input | Must be valid Signatures & RCDS as per the transaction implementation of the token spec. | Y        |
+
+#### Response:
+
+```json
+{  
+   "jsonrpc":"2.0",
+   "id":1,
+    "result":{
+        "txId":"80d87a8bd5cf2a3eca9037c2229f3701eed29360caa975531ef5fe476b1b70b5"
+    }
+}
+```
+
+
+
+
+
+## Daemon Methods
+
+### `get-daemon-tokens`:
```

Comment:
`get-tracked-tokens` would be a more explicit name.

Reply:
I like that :+1:

Reply:
What about the method to start tracking a new token?

Reply:
Currently calling a token method of an untracked token will begin tracking it. The API will respond with RPC Error Code `-32805 - Token Syncing` until it's synced and ready to serve requests. This suffices as an equivalent to the suggested call in my eyes, as you can just call a token method fire&forget for the same purpose. Thoughts?

Reply:
Ok I think it's good